README.md

```diff
@@ -44,6 +44,16 @@ opt
   * `args` - passed to `is_tuple_expired()` and `process_expired_tuple()` as additional context
   * `tuples_per_iteration` - number of tuples will be checked by one iteration
   * `full_scan_time` - time required for full index scan (in seconds)
+  * `iteration_delay` - max sleep time between iterations (in seconds)
+  * `full_scan_delay` - sleep time between full scans (in seconds)
+  * `on_full_scan_start` - call function on starting full scan iteration.
+    Receives `(task)` as arguments.
+  * `on_full_scan_complete` - call function on complete full scan iteration
+    Receives `(task)` as arguments.
+  * `on_full_scan_success` - call function on success full scan iteration
+    Receives `(task)` as arguments.
+  * `on_full_scan_error` - call function on error full scan iteration
+    Receives `(task, error)` as arguments.
```

Comment:
For me, it isn't obvious that `on_full_scan_complete` will be called after `on_full_scan_success` or `on_full_scan_error`. I propose to mention it.

Reply:
Done.